jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPart.java

```diff
@@ -126,9 +126,10 @@ public abstract static class Part implements Closeable
         private final String name;
         private final String fileName;
         private final HttpFields fields;
-        private Content.Source contentSource;
         private Path path;
-        private boolean temporary = true;
+        private final AutoLock lock = new AutoLock();
```

Comment:
Move this up close to the other `final` fields. I like initialized fields to be even before `final` fields that are initialized in the constructor.

Reply:
Done.